go/roothash/api/commitment/txnscheduler.go

```diff
@@ -11,7 +12,7 @@ import (
 
 // ProposedBatchSignatureContext is the context used for signing propose batch
 // dispatch messages.
-var ProposedBatchSignatureContext = signature.NewContext("oasis-core/roothash: proposed batch", signature.WithChainSeparation())
+var ProposedBatchSignatureContext = signature.NewContext("oasis-core/roothash: proposed batch", signature.WithChainSeparation(), signature.WithDynamicSuffix(" for runtime ", common.NamespaceHexSize))
```

Comment:
Maybe break it over multiple lines (also the other one).

Reply:
done :+1: